pkg/webui/console/views/application-link/index.js

```diff
@@ -154,38 +146,35 @@ class ApplicationLink extends React.Component {
         throw statsError
       }
     } catch (error) {
-      await this.setState({ error })
-      setSubmitting(false)
+      resetForm(values)
```

Comment:
Consider `setSubmitting(false)` instead in error cases. I believe it's not necessary to reset the form in these cases because it will disable the submit button until a field has been touched again.